scripts/preinstall

```diff
@@ -14,11 +14,11 @@ RESOURCE_FILES=(
 
 # Kill jamfHelper process to close all current notifications.
 echo "Killing any active jamfHelper notifications..."
-killall jamfHelper 2>/dev/null
+/usr/bin/killall jamfHelper 2>"/dev/null"
 
 # Clear out legacy package receipt (if present).
-if [[ $(pkgutil --pkgs | grep -q "com.elliotjordan.install_or_defer"; echo $?) -eq 0 ]]; then
-    pkgutil --forget "com.elliotjordan.install_or_defer"
+if [[ $(/usr/sbin/pkgutil --pkgs) == *"com.elliotjordan.install_or_defer"* ]]; then
```

Comment:
Good catch. The old way was overly complex.

Reply:
Turns out ShellCheck frowns upon logic built on exit codes. 😁